bricksrc/entity_properties.py

```diff
@@ -69,13 +69,19 @@
         RDFS.range: BRICK.CurrentFlowTypeShape,
     },
     BRICK.ratedPowerOutput: {
-        SKOS.definition: Literal("The rated power output of the entity"),
+        SKOS.definition: Literal("The maximum rated power output of the entity"),
         RDFS.range: BRICK.PowerOutputShape,
     },
     BRICK.measuredPowerOutput: {
-        SKOS.definition: Literal("The measured power output of the entity"),
+        SKOS.definition: Literal("The maximum measured power output of the entity"),
```

Comment:
Oh, good to know! I was reading some text that suggested that the rating was the maximum output rather than nominal, but I'm happy to revert the definition to be nominal power output

Reply:
I think for most practical purposes it's the maximum output, you're unlikely to see > 1000W/sqm with ambient temp < 25C for a normal installation, but I don't think it's technically correct.

Plus I'm assuming you'd want to reuse ratedPowerOutput and measuredPowerOutput for other entities?

Reply:
Agreed that reuse of those entity properties is a design goal.

If I'm understanding you, for solar panels "power output" is nominal rated power @ STC, but in other cases it is the maximum output?

Reply:
Not quite, power output is always the instantaneous measured output in W. But if the ambient conditions are identical to STC, power output will be equal to the rated power output.

Reply:
Let me know what you think of the current design; I amended the definition to indicate 'nominal' rather than maximum and added a `ambientTemperatureOfMeasurement` property which clarifies the context of the power output. There's also a `temperatureCoefficientofPmax` entity property now to provide additional metadata